db/schema.rb

```diff
@@ -612,9 +612,10 @@
     t.string   "control_device_number", limit: 255
     t.datetime "created_at",                        null: false
     t.datetime "updated_at",                        null: false
+    t.string   "description",           limit: 255
```

Comment:
I was thinking optional string like "NW door" or something--assuming just the two numbers aren't the most useful to a human